models/stg_zendesk__user.sql

```diff
@@ -31,8 +31,15 @@ final as (
         id as user_id,
         external_id,
         _fivetran_synced,
-        created_at,
-        updated_at,
+        {% if target.type == 'redshift' -%}
+            cast(last_login_at as timestamp without time zone) as last_login_at,
+            cast(created_at as timestamp without time zone) as created_at,
+            cast(updated_at as timestamp without time zone) as updated_at,
+        {% else -%}
+            last_login_at,
+            created_at,
+            updated_at,
```

Comment:
I updated the order of this!

Reply:
good eye @fivetran-reneeli, out of curiosity, @fivetran-joemarkiewicz for future reference, does the order in the `else` statement need to align with order of the `if` statement? I see how it absolutely could!

Reply:
The order doesn't matter for dbt to compile and execute, but would much rather have the order match to be consistent. Thanks so much team!